### PR TITLE
feat: A sentence with email

### PR DESCRIPTION
**Pull request by andrewlew1s**
So I put this lil' sentence at the end with an email address. I think it doesn't actually look too bad and I'm not sure we need a dedicated form at the moment. Perhaps if we get lots of users in the future.

We should think about what email we want to use and whether we should set one up like holisticweb@gmail.com

I also thought it might be sick to get mike@holistweb.com email addresses? Should we look into that?

Screen Shot 2020-03-23 at 2 23 08 PM

### CODE REVIEW

**Comment by michaelfitzhavey** [comment]
I like the idea of having `someone@holistic-web` email addresses, perhaps we can add that to the roadmap?

File: website/src/Website.vue [hunk 1]
```diff
@@ -19,6 +19,8 @@
 
 			</div>
 
+			<p>Feel free to contact us or report bugs at holisticweb@gmail.com</p>
```
Comment (michaelfitzhavey) [inline website/src/Website.vue]: Let's either use an email address that we have or set this email up before deploying this. We should also add a reference to our email to the main Readme so we remember to check it sometimes.

**Comment by michaelfitzhavey** [comment]
I don't know if I think this is enough to tick off the 'contact us' form but it is certainly a step in the right direction and our site will be better once we have this deployed. Great idea!

File: website/src/Website.vue (hunk 1 above)
Comment (andrewlew1s) [reply website/src/Website.vue]: I made: holisticweb.contact@gmail.com
I'll send you the password.

**Pull request closed without merging**